html_challenge1.html

```diff
@@ -0,0 +1,38 @@
+
+<!-- In the file you are going to copy and paste 
+the Sundial text below into your html file.
+```html -->
+
+<h1><strong>Sundial</strong></h1>
```

Comment:
Reminder: Just like code, you should indent if there is an html tag inside another. This makes it easier to read and know if there is a problem later.